handler.go

```diff
@@ -278,6 +295,9 @@ func (ca *Cagent) sendHeartbeat() error {
 	}
 	req = req.WithContext(ctx)
 	resp, err := ca.hubClient.Do(req)
+	if resp != nil && resp.StatusCode == 429 {
```

Comment:
429 can be replaced with http.StatusTooManyRequests

Reply:
Fixed